tables/output_tables.Rmd

```diff
@@ -99,16 +99,33 @@ histology_df %>%
     broad_histology_display
   )) %>%
   # remove doubled samples (WGS/RNA-seq etc.)
-  distinct(pbta_broad_histology, molecular_subtype, sample_id) %>%
-  # Count broad histology and subtype combinations
-  count(pbta_broad_histology, molecular_subtype) %>%
+  distinct(pbta_broad_histology, Kids_First_Participant_ID, molecular_subtype, sample_id) %>%
+  dplyr::rename(`Broad histology group` = pbta_broad_histology,
+         `OpenPBTA molecular subtype` = molecular_subtype)
+
+# Count broad histology and subtype combinations
+tumor_level <- subtype_n %>%
+  select(-Kids_First_Participant_ID) %>%
+  count(`Broad histology group`, `OpenPBTA molecular subtype`) %>%
+  dplyr::rename(Tumors = n)
+
+# Patient level
+pt_level <- subtype_n %>%
+  select(-sample_id) %>%
+  distinct() %>%
```

Comment:
So this works because the only remaining identifier in this data frame will be `Kids_First_Participant_ID`, correct?

Reply:
yes, sorry for the delay!